commands/lint.ts

```diff
@@ -54,6 +54,22 @@ export default class LintCommand extends BaseCommand {
     await this.lintRustFiles();
     console.log("Linting of Rust files complete.");
     console.log("");
+
+    await this.buildToolsImage("docker-tools");
+
+    console.log("");
+    console.log("Linting Docker files...")
+    console.log("");
+    const tmpDirectory = "dockerfiles-tmp/";
+    try {
+      await this.lintDockerFiles(tmpDirectory);
+      await this.cleanupAfterLintDockerFiles(tmpDirectory);
```

Comment:
@ryan-gang all of this logic (tmp, cleanup etc.) should just be inside `lintDockerfiles` - don't see why we're pulling it out here